src/witness/svcompSpec.ml

```diff
@@ -9,10 +9,13 @@ type t =
   | ValidFree
   | ValidDeref
   | ValidMemtrack
+  | MemorySafety (* Internal property for use in Goblint; serves as a summary for ValidFree, ValidDeref and ValidMemtrack *)
```

Comment:
This is fine for now. I think I promised somewhere to rework this myself anyway.